core/Plugin/Manager.php

```diff
@@ -1062,11 +1062,10 @@ public function getMissingPlugins()
         $missingPlugins = array();
 
         $plugins = $this->pluginList->getActivatedPlugins();
+
         foreach ($plugins as $pluginName) {
             // if a plugin is listed in the config, but is not loaded, it does not exist in the folder
-            if (!self::getInstance()->isPluginLoaded($pluginName)
-                && !$this->isPluginBogus($pluginName)
-            ) {
+            if (!$this->isPluginLoaded($pluginName) && !$this->isPluginBogus($pluginName) ) {
```

Comment:
No need to go via static container and load plugin manager like 90 times from there